nupic/algorithms/anomaly.py

```diff
@@ -157,6 +144,30 @@ def compute(self, activeColumns, predictedColumns,
     return score
 
 
+############################################
```

Comment:
@scottpurdy: didn't we do away with separators like `###############################` in the nupic coding conventions?